### DISCUSSION TITLE
[Show and Tell] Export zkSync Era transactions

### DISCUSSION THREAD

**Discussion by JackHamer09** [zkSync-Community-Hub/zksync-developers#178] [Show and Tell]
### Project Name

zkExport

### Repo Link

https://github.com/JackHamer09/zkExport

### Demo Link (Optional)

https://zkexport.netlify.app/

### Technologies Used

Vue, Nuxt 3, Block Explorer API

### Describe your project. What does it do? Why did you create it? Any interesting challenges you faced?

📢 **Introducing zkExport!** 🎉

Need to export your zkSync Era or Lite transactions? Look no further! With **zkExport**, you can easily export transaction data into formats like CSV, XLS, and more. Whether for study, taxes, verification, or record-keeping, zkExport has got you covered.

🌐 Available for both zkSync Era Mainnet & Testnet, as well as zkSync Lite Mainnet & Goerli Testnet.
📂 Multiple file formats supported, including CSV, XLSX, XML, ODS, HTML, TXT, and JSON.

Try it out: [zkExport.netlify.app](https://zkexport.netlify.app) 🚀

### What specific feedback are you looking for?

I just wanted to share this so that those in need of such a tool can find it.

### How can the community help or contribute to your project?

Share with your friends if you find this tool useful. Also feel free to contribute to the repo.

**Comment by ljx2099** [comment]
goog job man

**Comment by AsharibAli** [comment]
Awesome!

**Comment by daisbryce** [comment]
Great job on creating zkExport! It seems like a useful tool for users who need to export transaction data from zkSync Era or Lite. The support for multiple file formats and compatibility with both mainnet and testnet versions of zkSync is impressive.

I'll definitely check out the tool and share it with others who might find it useful. Keep up the good work, and if I encounter any issues or have suggestions for improvement while using zkExport, I'll be sure to contribute to the repository. Thanks for sharing this with the community! 🚀👍

**Comment by etesvipatdeb** [comment]
Awesome project! If you have any technical questions or need assistance with anything specific, feel free to ask here. Keep up the great work!

**Comment by alexmccullough** [comment]
The link for ZKsync Era seems to be broken. Is that tool still available somewhere?
